src/Microsoft.AspNetCore.Server.IISIntegration.Tasks/build/netstandard1.0/Microsoft.AspNetCore.Server.IISIntegration.Tasks.targets

```diff
@@ -0,0 +1,35 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <_AspNetCoreIISOldMSBuild Condition=" '$(MSBuildToolsVersion)' == '14.0' Or '$(MSBuildToolsVersion)' == '12.0' ">true</_AspNetCoreIISOldMSBuild>
```

Comment:
At the moment is a guardrail against assembly loading issues on the task. We might* be able to remove it, depending on things like https://github.com/Microsoft/msbuild/issues/1010.